tasks/main.yml

```diff
@@ -4,9 +4,27 @@
 - name: Add tsuru repositories.
   apt_repository: repo='ppa:tsuru/ppa' update_cache=yes
 
-- name: Install packages.
+- name: Install base packages.
   apt: update_cache=true name="{{ item }}"
-  with_items: packages
+  with_items:
+    - curl
+    - python-software-properties
+
+- name: Install latest version of Gandalf Server
+  apt: name=gandalf-server
+  when: gandalf_server_version is not defined
+
+- name: Install a specific version of Gandalf Server
+  apt: name=gandalf-server={{ gandalf_server_version }}
+  when: gandalf_server_version is defined
+
+- name: Install latest version of Archive Server
+  apt: name=archive-server
+  when: archive_server_version is not defined
+
+- name: Install a specific version of Archive Server
+  apt: name=archive-server={{ archive_server_version }}
+  when: archive_server_version is defined
```

Comment:
We talked about this offline and figured it was better to change. Martin did so.